amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/job/DispositionLifecycleJobExecuter.java

```diff
@@ -265,12 +268,6 @@ private void executeAction(final List<NodeRef> actionNodes)
                 }
                 Map<String, Serializable> props = Map.of(PARAM_NO_ERROR_CHECK, false);
 
-                if (freezeService.isFrozenOrHasFrozenChildren(parent.getParentRef()))
-                {
```

Comment:
Why is the log for the debug removed? This should be present to see what is the value during the debugging.

Reply:
In this If condition there will be not frozen cases, this condition is of no use then.